vagrant/Vagrantfile

```diff
@@ -63,35 +31,15 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
           target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['CONFIG_DIR']}", "/home/vagrant/opt/monorail/"
         end
 
-        if ENV['TEST_GROUP']		
-          if ENV['TEST_GROUP'] =~  /esxi-5-5-min-install.v2.0.test/ || ENV['TEST_GROUP'] =~  /esxi-5-5-max-install.v2.0.test/		
-            config.vm.provision "file", source: "./dhcpd.conf", destination: "~/dhcpd.conf"		
-            config.vm.provision "shell" do |s|		
-            s.inline = "cp /home/vagrant/dhcpd.conf /etc/dhcp"		
-            s.privileged = true		
-            end		
-          end		
-        end
         if ENV['PYTHON_REPOS']
-          config.vm.provision "file", source: "./rackhd-pm2-config.yml", destination: "~/rackhd-pm2-config.yml"
```

Comment:
If the yml is changed, we need to rebuild a box.
Maybe it's better to copy it during running.